scripts/base/frameworks/software/main.zeek

```diff
@@ -523,6 +526,11 @@ function found(id: conn_id, info: Info): bool
 	if ( ! info$force_log && ! addr_matches_host(info$host, asset_tracking) )
 		return F;
 
+	# This assumes that callers do not fill in info$ts, none of the current callers do.
+	if (info in found_cache)
```

Comment:
Nit.
```suggestion
	if ( info in found_cache )
```

Reply:
I wonder a bit if you want to provide `Cluster::node_down()` and/or `Cluster::node_up()` to clear out `found_cache` if a proxy restarts. That pattern exists in the known-scripts. With the assumption that proxies stay up (and if they are down, just for a short period of time), it might be better to not do that.

Reply:
There is a `info$force_log` attribute. Not sure if that should ignore the caching mechanism, or whether for its current usage that isn't relevant.

Reply:
I'm not sure if clearing out the cache is really required.. or at least, I'm not sure what it would accomplish.  The cache is short enough that a crashing proxy likely means there are bigger problems than a potentially missed software log entry.

I do want to further rework the data structure on the proxy side to better handle the version flip flops on that side as well.

force_log is only even used in one script and it doesn't look like there is a test for it.  I need to figure out why it uses it and what problem it solves.